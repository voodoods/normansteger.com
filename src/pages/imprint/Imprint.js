import React, { Component } from 'react'
import PropTypes from 'prop-types'
import StyledImprint from './Imprint.styled'

class Imprint extends Component {
  render() {
    return (
      <StyledImprint className="Imprint pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1">
        <div>
          <h3>
            <a name="imprint">Imprint</a>
          </h3>
          <p>All informations according to ยง 5 TMG</p>
          <p>
            Norman Steger <br />
            Steinweg 26a<br />
            93059 Regensburg <br />
          </p>
          <p>
            <b>Represented by: </b>
            <br />
            Norman Steger<br />
          </p>
          <p>
            <b>Contact:</b> <br />
            eMail: <a href="mailto:me@normansteger.com">me@normansteger.com</a>
            <br />
          </p>
          <p>
            <br />
            <h3>Disclaimer: </h3>
            <br />
            <b>Cookies and Data Tracking</b>
            <br />
            <br />
            Because I have no interest in tracking data that is directly or indirectly related to you as a person or
            your behavior on this website I will take care to the best of my abilities not to use any kind of
            technologies that allow the tracking or storage of personal or usage related data on this website. This also
            means that I will take immediate action to remove such technologies from this website should they ever
            become part of it without my knowledge.
            <br />
            <br />
            <b>Liability for external Links / References</b>
            <br />
            <br />
            Our offer contains links to external websites of third parties whose contents I have no influence on.
            Therefore, I can not assume any liability for these external contents. Responsibility for the contents of
            the linked Pages lies with their respective provider or operator. The linked pages were checked for possible
            legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A
            permanent content control of the linked pages is but without concrete evidence of an infringement
            unreasonable. Upon becoming aware of Violations, I will remove such links immediately.<br />
            <br />
          </p>
          <br />
        </div>
      </StyledImprint>
    )
  }
}

Imprint.propTypes = {
  location: PropTypes.shape({
    pathname: PropTypes.string,
  }),
}

export default Imprint
