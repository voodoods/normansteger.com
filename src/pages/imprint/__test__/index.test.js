import React from 'react'
import { shallow } from 'enzyme'
import Imprint from '../'

describe('<Imprint /> component index', () => {
  it('renders without crashing', () => {
    const props = { location: { pathname: '/imprint/' } }
    const wrapper = shallow(<Imprint {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
