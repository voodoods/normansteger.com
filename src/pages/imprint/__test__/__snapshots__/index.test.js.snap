// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Imprint /> component index renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Imprint
    location={
      Object {
        "pathname": "/imprint/",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div>
        <h3>
          <a
            name="imprint"
          >
            Imprint
          </a>
        </h3>
        <p>
          <b>
            This website is owned and operated by: 
          </b>
          <br />
          Norman Steger
          <br />
        </p>
        <p>
          <b>
            Contact:
          </b>
           
          <br />
          eMail: 
          <a
            href="mailto:me@normansteger.com"
          >
            me@normansteger.com
          </a>
          <br />
        </p>
        <p>
          <br />
          <h3>
            Disclaimer: 
          </h3>
          <b>
            Purpose of this site
          </b>
          <br />
          <br />
          This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.
          <br />
          <br />
          <b>
            Cookies and Data Tracking
          </b>
          <br />
          <br />
          Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.
          <br />
          <br />
          <b>
            Liability for external Links / References
          </b>
          <br />
          <br />
          Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.
          <br />
          <br />
        </p>
        <br />
      </div>,
      "className": "Imprint pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            <a
              name="imprint"
            >
              Imprint
            </a>
          </h3>,
          <p>
            <b>
              This website is owned and operated by: 
            </b>
            <br />
            Norman Steger
            <br />
          </p>,
          <p>
            <b>
              Contact:
            </b>
             
            <br />
            eMail: 
            <a
              href="mailto:me@normansteger.com"
            >
              me@normansteger.com
            </a>
            <br />
          </p>,
          <p>
            <br />
            <h3>
              Disclaimer: 
            </h3>
            <b>
              Purpose of this site
            </b>
            <br />
            <br />
            This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.
            <br />
            <br />
            <b>
              Cookies and Data Tracking
            </b>
            <br />
            <br />
            Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.
            <br />
            <br />
            <b>
              Liability for external Links / References
            </b>
            <br />
            <br />
            Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.
            <br />
            <br />
          </p>,
          <br />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              name="imprint"
            >
              Imprint
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Imprint",
              "name": "imprint",
            },
            "ref": null,
            "rendered": "Imprint",
            "type": "a",
          },
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <b>
                This website is owned and operated by: 
              </b>,
              <br />,
              "Norman Steger",
              <br />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This website is owned and operated by: ",
              },
              "ref": null,
              "rendered": "This website is owned and operated by: ",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "Norman Steger",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <b>
                Contact:
              </b>,
              " ",
              <br />,
              "eMail: ",
              <a
                href="mailto:me@normansteger.com"
              >
                me@normansteger.com
              </a>,
              <br />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Contact:",
              },
              "ref": null,
              "rendered": "Contact:",
              "type": "b",
            },
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "eMail: ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "me@normansteger.com",
                "href": "mailto:me@normansteger.com",
              },
              "ref": null,
              "rendered": "me@normansteger.com",
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <br />,
              <h3>
                Disclaimer: 
              </h3>,
              <b>
                Purpose of this site
              </b>,
              <br />,
              <br />,
              "This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.",
              <br />,
              <br />,
              <b>
                Cookies and Data Tracking
              </b>,
              <br />,
              <br />,
              "Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.",
              <br />,
              <br />,
              <b>
                Liability for external Links / References
              </b>,
              <br />,
              <br />,
              "Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.",
              <br />,
              <br />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Disclaimer: ",
              },
              "ref": null,
              "rendered": "Disclaimer: ",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Purpose of this site",
              },
              "ref": null,
              "rendered": "Purpose of this site",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Cookies and Data Tracking",
              },
              "ref": null,
              "rendered": "Cookies and Data Tracking",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Liability for external Links / References",
              },
              "ref": null,
              "rendered": "Liability for external Links / References",
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
      ],
      "type": "div",
    },
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  padding: 6em;
  width: 80%;
  h3 {
    font-size: 1.5em;
  }

  @media (max-width: 1024px) {
    padding: 1em;
    width: 100%;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div>
          <h3>
            <a
              name="imprint"
            >
              Imprint
            </a>
          </h3>
          <p>
            <b>
              This website is owned and operated by: 
            </b>
            <br />
            Norman Steger
            <br />
          </p>
          <p>
            <b>
              Contact:
            </b>
             
            <br />
            eMail: 
            <a
              href="mailto:me@normansteger.com"
            >
              me@normansteger.com
            </a>
            <br />
          </p>
          <p>
            <br />
            <h3>
              Disclaimer: 
            </h3>
            <b>
              Purpose of this site
            </b>
            <br />
            <br />
            This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.
            <br />
            <br />
            <b>
              Cookies and Data Tracking
            </b>
            <br />
            <br />
            Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.
            <br />
            <br />
            <b>
              Liability for external Links / References
            </b>
            <br />
            <br />
            Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.
            <br />
            <br />
          </p>
          <br />
        </div>,
        "className": "Imprint pure-u-lg-3-5 pure-u-md-1 pure-u-sm-1",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              <a
                name="imprint"
              >
                Imprint
              </a>
            </h3>,
            <p>
              <b>
                This website is owned and operated by: 
              </b>
              <br />
              Norman Steger
              <br />
            </p>,
            <p>
              <b>
                Contact:
              </b>
               
              <br />
              eMail: 
              <a
                href="mailto:me@normansteger.com"
              >
                me@normansteger.com
              </a>
              <br />
            </p>,
            <p>
              <br />
              <h3>
                Disclaimer: 
              </h3>
              <b>
                Purpose of this site
              </b>
              <br />
              <br />
              This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.
              <br />
              <br />
              <b>
                Cookies and Data Tracking
              </b>
              <br />
              <br />
              Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.
              <br />
              <br />
              <b>
                Liability for external Links / References
              </b>
              <br />
              <br />
              Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.
              <br />
              <br />
            </p>,
            <br />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                name="imprint"
              >
                Imprint
              </a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Imprint",
                "name": "imprint",
              },
              "ref": null,
              "rendered": "Imprint",
              "type": "a",
            },
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <b>
                  This website is owned and operated by: 
                </b>,
                <br />,
                "Norman Steger",
                <br />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "This website is owned and operated by: ",
                },
                "ref": null,
                "rendered": "This website is owned and operated by: ",
                "type": "b",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "Norman Steger",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <b>
                  Contact:
                </b>,
                " ",
                <br />,
                "eMail: ",
                <a
                  href="mailto:me@normansteger.com"
                >
                  me@normansteger.com
                </a>,
                <br />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Contact:",
                },
                "ref": null,
                "rendered": "Contact:",
                "type": "b",
              },
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "eMail: ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "me@normansteger.com",
                  "href": "mailto:me@normansteger.com",
                },
                "ref": null,
                "rendered": "me@normansteger.com",
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <br />,
                <h3>
                  Disclaimer: 
                </h3>,
                <b>
                  Purpose of this site
                </b>,
                <br />,
                <br />,
                "This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.",
                <br />,
                <br />,
                <b>
                  Cookies and Data Tracking
                </b>,
                <br />,
                <br />,
                "Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.",
                <br />,
                <br />,
                <b>
                  Liability for external Links / References
                </b>,
                <br />,
                <br />,
                "Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.",
                <br />,
                <br />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Disclaimer: ",
                },
                "ref": null,
                "rendered": "Disclaimer: ",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Purpose of this site",
                },
                "ref": null,
                "rendered": "Purpose of this site",
                "type": "b",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "This website is for sole informational purposes about its representative as a professional. It does not offer any sort of commercial services or products nor does it follow any commercial purposes in any way.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Cookies and Data Tracking",
                },
                "ref": null,
                "rendered": "Cookies and Data Tracking",
                "type": "b",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "Because I have no interest in tracking data that is directly or indirectly related to you as a person or your behavior on this website I will take care to the best of my abilities not to use any kind of technologies that allow the tracking or storage of personal or usage related data on this website. This also means that I will take immediate action to remove such technologies from this website should they ever become part of it without my knowledge.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Liability for external Links / References",
                },
                "ref": null,
                "rendered": "Liability for external Links / References",
                "type": "b",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "Our offer contains links to external websites of third parties whose contents I have no influence on. Therefore, I can not assume any liability for these external contents. Responsibility for the contents of the linked Pages lies with their respective provider or operator. The linked pages were checked for possible legal violations at the time of linking. Illegal contents were to Time of linking not recognizable. A permanent content control of the linked pages is but without concrete evidence of an infringement unreasonable. Upon becoming aware of Violations, I will remove such links immediately.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
        ],
        "type": "div",
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  padding: 6em;
  width: 80%;
  h3 {
    font-size: 1.5em;
  }

  @media (max-width: 1024px) {
    padding: 1em;
    width: 100%;
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
